web-app/css/openboxes.css

```diff
@@ -1191,7 +1191,9 @@ button.action-btn, button.previous, button.next {
 /* temporarily hide the nav bar until we figure out whether we want to keep it or not */
 .nav { display: none; }
 
-li.ui-menu-item { text-align: left; font-weight: bold; }
+ul.ui-autocomplete { width: auto !important; }
```

Comment:
These two `importants` scare me a little bit. Maybe it would be better to add an additional class to it?